west.yml

```diff
@@ -33,6 +33,8 @@ manifest:
       url-base: https://github.com/ThrowTheSwitch
     - name: armmbed
       url-base: https://github.com/ARMmbed
+    - name: thomasstenersen
```

Comment:
@thomasstenersen Can you remove those references to your private repo so we can merge this?

Reply:
The nrfxlib repo must also be updated as the revision pointed by the PR does not fix known issues.
Just a reminder :)